docs/data/material/pages.ts

```diff
@@ -131,6 +131,11 @@ const pages = [
       {
         pathname: '/x/data-grid/',
         subheader: 'data-grid',
+        children: [
+          { pathname: '/x/react-data-grid/', title: 'Overview' },
+          { pathname: '/x/react-data-grid/demo/', title: 'Demo' },
+          { pathname: '/x/react-data-grid/getting-started/', title: 'Getting Started' },
+        ],
```

Comment:
Add some data grid links for the people who used to see Data Grid in material docs sidebar.

<img width="279" alt="Screen Shot 2565-01-21 at 14 07 52" src="https://user-images.githubusercontent.com/18292247/150490593-62ccf058-5ff0-455c-a1b2-ea2e04534595.png">